### DISCUSSION TITLE
Pinned split tab

### DISCUSSION THREAD

**Discussion by Axenide** [zen-browser/desktop#1751] [Ideas]
Hi, I think it would be a really useful feature to have the ability to pin a split tab, so you can have it fixed on one side while browsing normally on the other.

While this is basically like having the Zen Sidebar open, this would be useful for more temporary things, like having a YouTube video on one side, or a Twitch chat.

It would make the tiling feature more powerful for custom workflows.

An alternative would be being able to switch tabs inside the currently focused split, so you keep your layout.

**Comment by sai-educ** [comment]
I agree with the utility of this feature. I miss it on Zen but this is how Arc does it - 

<img width="266" alt="SCR-20240927-nriq-2" src="https://github.com/user-attachments/assets/21e55faf-3f92-4d8f-914a-ffd7962aaad9">
